code/modules/research/designs/autolathe_designs.dm

```diff
@@ -785,11 +785,11 @@
 	category = list("hacked", "Security")
 
 /datum/design/a357
-	name = "Ammo Box (.357)"
+	name = "\improper .357 bullet"
```

Comment:
I dont think `\improper` will actually do anything here.

Reply:
This will be resolved in the next commit as it removes printing individual bullets in favour of a new ammo box which cannot be directly loaded into the autoloader or .357.